packages/jest-core/src/collectHandles.ts

```diff
@@ -125,6 +145,14 @@ export default function collectHandles(): HandleCollectionResult {
     // callback, we will not yet have seen the resource be destroyed here.
     await asyncSleep(100);
 
+    if (activeHandles.size !== 0) {
```

Comment:
```suggestion
    if (activeHandles.size > 0) {
```

I find it more readable 🙂